files/templates/alertmanager/alertmanager.yaml

```diff
@@ -180,6 +188,14 @@ receivers:
       url: '{{ template "__alert_silence_link" .}}'
       style: '{{ if eq .Status "firing" }}danger{{ else }}default{{ end }}'
 
+- name: team_teddyfriends_slack
+  slack_configs:
+  - channel: '#team-teddyfriends'
+    actions:
```

Comment:
```suggestion
    send_resolved: true
    actions:
```

Reply:
I always thought this changed the color on slack to green but let's keep it that way for now then